hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/v2/pages/overview/components/SummaryCardGroup.tsx

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import OverviewSummaryCard from "@/v2/components/overviewCard/overviewSummaryCard";
+import { getTaskIndicatorIcon } from "@/v2/pages/overview/overview.utils";
+import { Col } from "antd";
+import filesize from "filesize";
+import React from "react";
+
+type OverviewSummaryCardGroupProps = {
+  loading?: boolean;
+  openSummarytotalRepSize: number;
+  openSummarytotalUnrepSize: number;
+  openSummarytotalOpenKeys: number;
+  deletePendingSummarytotalRepSize: number;
+  deletePendingSummarytotalUnrepSize: number;
+  deletePendingSummarytotalDeletedKeys: number;
+  taskRunning: boolean;
+  taskStatus: boolean;
+}
+
+const size = filesize.partial({ round: 1 });
+
+const getSummaryTableValue = (
+  value: number | string | undefined,
+  colType: 'value' | undefined = undefined
+): string => {
+  if (!value) return 'N/A';
+  if (colType === 'value') return String(value as string)
+  return size(value as number)
+}
+
+const OverviewSummaryCardGroup: React.FC<OverviewSummaryCardGroupProps> = ({
+  loading = true,
+  openSummarytotalRepSize,
+  openSummarytotalUnrepSize,
+  openSummarytotalOpenKeys,
+  deletePendingSummarytotalRepSize,
+  deletePendingSummarytotalUnrepSize,
+  deletePendingSummarytotalDeletedKeys,
+  taskRunning,
+  taskStatus
+}) => {
+
+  const taskIcon: React.ReactElement = getTaskIndicatorIcon(taskRunning, taskStatus);
```

Comment:
For open keys and delete pending keys also, data correctness relies on `OmTableInsightTask`.